### DISCUSSION TITLE
Add a hotkey option to toggle the tab bar (whether its vertical or horizontal)

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#569] [Feature Request]
I have vertical tabs enabled and sometimes I want to hide them so web pages can take up the full width of my screen. A hotkey should be added called "Toggle tab bar (vertical or horizontal)" and rename "Show all tabs panel" to "Show tabs dropdown." I thought panels refer to the whole tab bar and not the dropdown menu.

**Comment by ioc2e3** [comment]
Would also like a hotkey option like this...
